test/manual/api/system_api_legacy_tests.sh

```
@@ -0,0 +1,531 @@
+    echo "0"
+  else
+    echo "$list" | wc -w
+  fi
+}
+
+# Parse JSON response with error handling
+parse_response() {
+  local raw="$1"
+  
+  # Skip HTTP headers and get the JSON body
+  local json_body=$(echo "$raw" | sed '1,/^\s*$/d')
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)